Doc/library/sqlite3.rst

```diff
@@ -85,7 +85,9 @@ At this point, our database only contains one row::
 The result is a one-item :class:`tuple`:
 one row, with one column.
 Now, let us insert three more rows of data,
-using :meth:`~Cursor.executemany`::
+using :meth:`~Cursor.executemany`.
+We use :ref:`question mark placeholders <sqlite3-placeholders>`
+to bind ``data`` to the SQL statement::
```

Comment:
I think the part about using placeholders rather than string formatting was a pretty important thing that anyone using `sqlite3` should be aware of, and this move makes it a lot easier to miss.

It would be better to replace this sentence with a short paragraph -- possibly after the example -- that:
* points out that placeholders were used instead of string formatting;
* warns the user against using string formatting;
* explicitly links to the how-to (e.g. "For more information about placeholders, see :ref:`...`.")

Reply:
Yes, I think you are right; we should point out the danger of injection attacks. I have an idea of how to phrase it; let me sleep on it.

Reply:
CAM, Ezio: Please take a new look.

I think it worked out ok. I'm not sure about the string formatting link, but I think it is ok since this is a tutorial. I also added a wikipedia link to SQL injection attacks.